README.md

```diff
@@ -40,12 +40,20 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 - [ ] What is the purpose of using _sessions_?
 
+To store info info about a current logged in user. It can hold a cookie to show they are logged in, or things relevant to the site like videos based on recent searches. 
+
 - [ ] What does bcrypt do to help us store passwords in a secure manner.
 
+It hashes them to generate a new version of the password that cannot be used if someone grabs that version.
+
```

Comment:
something I would have liked to hear is how it takes a plaintext password and applies it to a hash function to encrypt it so  it would take awhile to hack.